transport/src/main/java/io/netty/channel/ChannelConfig.java

```diff
@@ -234,6 +236,8 @@
     int getWriteBufferLowWaterMark();
```

Comment:
@normanmaurer  why need to add deprecated for getWriteBufferLowWaterMark(). if so, I need add new method such as getWriteBufferWaterMark(), 

now I can't make sure is there need to add the new method: ChannelConfig getWriteBufferWaterMark().
if I add one, I need add one member such as writeBufferWaterMark into DefaultChannelConfig, but if I do so, the old set method of setWriteBufferHighWaterMark will change the writeBufferWaterMark's field so that the final MUST be break expect remove the final keywords.

if consider the old methods' support. now I only need these methods and I implement it just now:

/**
 * @deprecated Use {@link #setWriteBufferWaterMark(WriteBufferWaterMark)}
 */
ChannelConfig setWriteBufferHighWaterMark(int writeBufferHighWaterMark);

/**
 * @deprecated Use {@link #setWriteBufferWaterMark(WriteBufferWaterMark)}
 */
ChannelConfig setWriteBufferLowWaterMark(int writeBufferLowWaterMark);

int getWriteBufferHighWaterMark();
int getWriteBufferLowWaterMark();

//new add method
ChannelConfig setWriteBufferWaterMark(WriteBufferWaterMark writeBufferWaterMark);